src/main/java/spoon/reflect/declaration/CtAnnotation.java

```diff
@@ -81,6 +84,18 @@
 	@PropertyGetter(role = VALUE)
 	<T extends CtExpression> T getValue(String key);
 
+	/** Returns the actual value of an annotation property */
+	@DerivedProperty
+	Object getValueAsObject(String key);
+
+	/** Returns the actual value of an annotation property, as an integer (utility method) */
+	@DerivedProperty
+	int getValueAsInt(String key);
```

Comment:
What if the key does not exist? We would get a ClassCastException?

Reply:
It would return null. In Java cast(null) returns null.

Reply:
It will work with the string but not int as an int can never return null.
Actually I just tested it and it throws a NPE. Could you add a check?

Reply:
done